src/sum_terms.py

```diff
@@ -0,0 +1,14 @@
+"""Implementation of Sum of nth Term Kata on Code Wars."""
+from __future__ import division
+
+
+def series_sum(n):
+    """Return nth value in series."""
+    if n <= 1:
+        return "%.2f" % n
+    else:
+        count = 1
+        for num in range(n - 1):
+            n = n - 1
```

Comment:
so I tried to use the backward count but I just couldn't implement it, it was messing the math for some reason.